lib/libutee/amble/Apache License.txt

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
Should we have some words in `LICENSE` file about this apache license? @jbech-linaro